project/README.md

```diff
@@ -1,12 +1,31 @@
+## You are here
 Main location for reproducibility study.
 
-Layout is as below (subject to change):
+## How to use
+All commands are intended to be run from this location with the `mosdef-study38` environment active.
```

Comment:
Let’s add the bash command to activate the conda env here. Since it is the only time we will need to list it.

Reply:
you got it! I actually added it in my most recent commits when I updated with instructions for #29